wfe/web-front-end.go

```diff
@@ -86,6 +86,8 @@ type problem struct {
 func sendError(response http.ResponseWriter, message string, code int) {
 	problem := problem{Detail: message}
 	problemDoc, err := json.Marshal(problem)
+  // XXX Double check that this is not redundant with the behavior of http.Error
```

Comment:
I was saying that I didn't know whether http.Error might set <tt>Content-Type: application/problem+json</tt>, though I thought it was unlikely since that seems to be an obscure REST practice.